journalbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,185 @@
+
+include::../../libbeat/docs/shared-download-and-install.asciidoc[]
+
+[[deb]]
+*deb:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {beatname_uc} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-amd64.deb
+sudo dpkg -i {beatname_lc}-{version}-amd64.deb
+------------------------------------------------
+
+endif::[]
+
+[[rpm]]
+*rpm:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {beatname_uc} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-x86_64.rpm
+sudo rpm -vi {beatname_lc}-{version}-x86_64.rpm
+------------------------------------------------
+
+endif::[]
+
+[[linux]]
+*linux:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {beatname_uc} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-linux-x86_64.tar.gz
+tar xzvf {beatname_lc}-{version}-linux-x86_64.tar.gz
+------------------------------------------------
+
+endif::[]
+
+[id="{beatname_lc}-configuration"]
+=== Step 2: Configure {beatname_uc}
+
+Before running {beatname_uc}, you need to specify the location of the systemd
+journal files and configure how you want the files to be read.
+
+include::../../libbeat/docs/shared-configuring.asciidoc[]
+
+Here is a sample of the +{beatname_lc}+ section of the +{beatname_lc}.yml+ file.
+{beatname_uc} uses predefined default values for most configuration options.
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+journalbeat.inputs:
+- paths: ["/path/to/journal/directory"]
+  seek: head
+----------------------------------------------------------------------
+
+To configure {beatname_uc}:
+
+. Specify a list of paths to your systemd journal files. Each path can be a
+directory path (to collect events from all journals in a directory), or a file
+path. For example:
++
+["source","sh",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- paths:
+  - "/dev/log"
+  - "/var/log/messages/my-journal-file.journal"
+----
++
+If no paths are specified, {beatname_uc} reads from the default journal.
+
+. Set the <<{beatname_lc}-seek,`seek`>> option to control the position where
+{beatname_uc} starts reading the journal. The available options are `head`,
+`tail`, and `cursor`. The default is `cursor`, which means that
+{beatname_uc} will continue reading where it left off after a reload or restart.
```

Comment:
The `seek` setting is likely one of the most important ones to understand how the input works.

For each of these values:
- What happens if journalbeat is started the first time? Will it collect existing logs or only new logs?
- What happens on journalbeat restart?
  - Will it resend all logs?
  - Will it continue from the last known position?
  - Will logs get lost between restart?

Q: When using cursor, do we collect old logs as well? Some users might prefer to send all logs when using cursor, others might prefer to send only new logs. How are the different behaviors configured? E.g. `tail` -> restart with `cursor`?

Reply:
If `cursor` is configured for seeking and there is no cursor available for that input, journalbeat starts to read it from the beginning.
If someone wants to start from the end of the journal he/she can start with `tail` and then switch to `cursor` and restart the Beat. I thought about adding a configuration option so the user can specify whether, journalbeat seek to tail or head. But felt unnecessary. Ofc, if needed, I can add it easily.

Reply:
I don't want to cover all of this info in the getting started, so I'll make the link to the reference docs more obvious, and I'll add some stuff to the reference topic.